test/config_thunks.tests.js

```diff
@@ -441,4 +454,27 @@ describe('ReduxEntityConfig - thunks', () => {
       });
     });
   });
+
+  describe('unsuccessful call with no parseServerErrorsFunc passed to config block', () => {
+    afterEach(() => restoreSpies());
+
+    const updateFunc = () => Promise.reject(standardError);
+    const config = new ReduxConfig({
+      entityName: 'users',
+      schema: schemas.USERS,
+      updateFunc,
+    });
+
+    it('returns the unformatted result as the errors response', () => {
+      const mockStore = reduxMockStore(store);
+
+      return mockStore.dispatch(config.actions.update())
+        .catch((response) => {
+          const dispatchedActions = mockStore.getActions();
+          const updateFailureAction = find(dispatchedActions, { type: 'users_UPDATE_FAILURE' });
+
+          expect(updateFailureAction.payload).toEqual({ errors: response });
```

Comment:
Should this expectation expect that the payload is `standardError` (The return value from the update API call)?

```js
expect(updateFailureAction.payload).toEqual(standardError);
```